src/keymap_key.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013-2015, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use gdk_sys;
+use glib::translate::*;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+#[repr(C)]
+pub struct KeymapKey {
```

Comment:
Why not make it a tuple struct around `ffi::GdkKeyMapKey`? Then we don't have to duplicate the struct layout here

Reply:
Still a question :)